docs/full-node/run-a-full-terra-node/sync.mdx

```diff
@@ -6,26 +6,134 @@ import Admonition from '@theme/Admonition';
 
 # Sync
 
-Use this guide to [sync the chain from from genesis](#sync-from-genesis) or use a [snapshot](#sync-from-snapshot) for a quicker sync. 
+This guide will walk you through how to [sync the chain from genesis](#sync-from-genesis) or use a [snapshot](#sync-from-snapshot) for a quicker sync. 
 
 ## Sync from genesis
 
-After [choosing a network](./join-a-network.mdx#join-a-public-network), start the sync and check that everything is running smoothly:
+The process for syncing your full node from genesis will differ between the Pisco testnet and Phoenix-1 mainnet networks. You may proceed with the instructions corresponding with your [chosen network](./join-a-network.mdx#join-a-public-network) below.
 
-```bash
+### Pisco testnet
```

Comment:
I would number all of the substeps in this section using an ordered list. It helps readers remember what step they left off on. Same for the Pheonix section.